modules/ROOT/pages/dataweave-cookbook-extract-data.adoc

```diff
@@ -504,12 +461,11 @@ output application/json
 ----
 
 [[key_value]]
-== Key-Value Pair Selector (`.&keyName`)
-
+== Key-Value Pair Selector (`.&myKey`)
 
-The `&` selector retrieves both the keys and the values of all matching keys pairs
-in the current context. These are returned as an object, containing these
-retrieved keys and values.
+The `&amp;` selector acts on arrays and objects. `&amp;` retrieves both the
+keys and the values of all matching keys pairs in the current context. These
+are returned as an object, containing these retrieved keys and values.
```

Comment:
"the retrieved keys and values" sounds better.

Reply:
fixed